quiz.js

```diff
@@ -17,9 +17,11 @@ db.dropDatabase();
 
 // your code begins here
 
+db.westeros.insert({house: 'Arryn', motto: "As High as Honor"});
+db.westeros.insert({house: 'Stark', motto: "Winter is Coming"});
+db.westeros.insert({house: 'Targaryen', motto: "Fire and Blood"});
 
```

Comment:
Close but no cigar. The syntax is fine, but the instructions say that the collection should be called 'houses'. Plus, since our database is also called 'westeros' we would probably also want to avoid confusion by giving our collection a different name.